es/es-server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -5083,10 +5083,11 @@ public void testRebuildLocalCheckpointTracker() throws Exception {
         final AtomicLong globalCheckpoint = new AtomicLong(SequenceNumbers.NO_OPS_PERFORMED);
         Path translogPath = createTempDir();
         int numOps = scaledRandomIntBetween(1, 500);
+        int maxDocId = 2 * numOps;
```

Comment:
Why the `* 2` ?

Reply:
In ES master:

```final int maxIdValue = randomInt(numOps * 2);```

they use a method called generateHistoryOnReplica that limits the max doc id to twice the number of ops (versus our limit of 100, which falls short of the max num of ops going up to 500)